File: data-pipelines/elt/elt-job.py
import json
from pathlib import Path

def run_elt():
  raw_dir = Path("data/raw")
  dw_dir = Path("data/processed/evaluation")
  dw_dir.mkdir(parents=True, exist_ok=True)

  for path in raw_dir.rglob("*.jsonl"):
    out = dw_dir / path.name
    out.write_text(path.read_text(), encoding="utf-8")

if __name__ == "__main__":
  run_elt()

File: data-pipelines/elt/elt-job.ts
import { readFileSync, writeFileSync, mkdirSync } from 'fs';
import { join } from 'path';

export function runElt() {
  const rawDir = join('data', 'raw');
  const outDir = join('data', 'processed', 'evaluation');
  mkdirSync(outDir, { recursive: true });

  // simple passthrough for demo
  const file = join(rawDir, 'raw-conversations.jsonl');
  const out = join(outDir, 'eval.jsonl');
  writeFileSync(out, readFileSync(file));
}
